### DISCUSSION TITLE
How do I find the config.yml file?

### DISCUSSION THREAD

**Discussion by opticaldrive** [community/community#60961] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I find the config.yml file?
https://docs.github.com/en/pages/quickstart
I followed that until the part where it told me to change the config.jml file.
I can't find it in my repository.
Could someone help?
Thank you.

**Comment by lshigami** [comment, marked as answer]
The config.yml file is a configuration file for GitHub Pages sites. It is used to specify the name of the site, the theme, and other settings. The file is located in the root directory of your GitHub Pages repository.

If you are using the default GitHub Pages setup, the config.yml file will be created for you automatically. However, if you have created your own GitHub Pages site, you may need to create the file manually.

To create the config.yml file, open a text editor and create a new file called config.yml. Then, add the following content to the file:

name: My Site
theme: minima
This will create a simple GitHub Pages site with the minima theme.

Once you have created the config.yml file, you can commit it to your repository and push it to GitHub. GitHub Pages will then use the file to configure your site.

Here are the steps on how to find the config.yml file:

Go to your GitHub Pages repository.
Click on the Code tab.
In the Files section, look for the config.yml file.
If you cannot find the config.yml file, you can create it manually by following the instructions above.

**Reply by opticaldrive** [reply]
Thank you.
Will try.

**Reply by opticaldrive** [reply]
In the docs it says _config.yml not config.yml is this a typo?

**Reply by opticaldrive** [reply]
I don't know if I created the file properly.

**Reply by radar-blip** [reply]
`_config.yml` is correct.

**Reply by opticaldrive** [reply]
Should I rename the file to that then?

**Reply by radar-blip** [reply]
Yes.

**Reply by opticaldrive** [reply]
Thank you!
Super fast Answer!

**Reply by opticaldrive** [reply]
https://github.com/V205Arduino/V205Arduino.github.io
Is this the correct way?
Thank you.

**Reply by radar-blip** [reply]
It is.
You're welcome!

**Reply by opticaldrive** [reply]
Thanks to everyone!
also what themes are available for GitHub pages?
Thanks!

**Reply by JamesNewton** [reply]
This is very confusing. In your question and answer, you talk about config.yml, but in the actual repo, and in the quickstart page, there is an underscore before the config.yml. Is this discussion site just pulling off the config.

**Reply by opticaldrive** [reply]
Sorry @JamesNewton  I don’t think I understand your question. :(

**Reply by JamesNewton** [reply]
@V205Arduino What see above on the answer from @lshigami is this:
![image](https://github.com/community/community/assets/419392/b3e6eb66-5019-49e4-b5d8-6b4a22aae4f7)
do you see that there is no underscore in front of the "config.yml"?

But in the actual quickstart page, I see this:
![image](https://github.com/community/community/assets/419392/2d14b26f-5884-4d6e-8ba2-d464f12d2bec)
do you se that there IS an underscore in front of the "config.yml"?

Which is it?

**Reply by opticaldrive** [reply]
I noticed both.

**Reply by SoninLeandro** [reply]
Hi guys, is this the file I'm suppose to use to add a url to redirect my site from githubpages to my own domain?

**Reply by opticaldrive** [reply]
@SoninLeandro Sorry, I have no idea

**Reply by opticaldrive** [reply]
https://docs.github.com/en/rest/pages/pages?apiVersion=2022-11-28 Maybe something here?

**Comment by jshtuazon2** [comment]
51984344

**Comment by jshtuazon2** [comment]
Joshua Tuazon Fontamillas
09683047330

**Comment by jneea-new** [comment]
I would also be interested in finding the default `_config.yml` file that appears to be **normally** copied to the root directory when you activate github pages (but that wasn't done for me: instead, only an `index.html` file was created there).

The docs give the three lines to **add** but when I tried to *create* the config file with just these 3 lines, it didn't work.  (I get "deploy errors".) And I searched all the docs but couldn't find the contents of that file as it is supposed to be created. 

Thanks in advance for your constructive help!

**Comment by GrumpyCarebear** [comment]
Hi, I already added the file and committed, when I go to my webpage it is still showing my original title and my new title. I am not sure what I am doing wrong. the _config.yml was added to main. when I committed I didn't add it to a new branch. What can I do to change my title to reflect the new changes that I want. Thank you
